### ISSUE TITLE
Weird actions in Robotwin2 Evaluation

### ISSUE THREAD

**Issue by CuriousCat-7** [starVLA/starVLA#187]
Hello, thank you for your code.

I followed the https://github.com/starVLA/starVLA/blob/starVLA/examples/Robotwin/README.md
And run the server from the starVLA pretrained Qwen3-VL-OFT-Robotwin2 model:
```bash
export HF_ENDPOINT=https://hf-mirror.com
hf download Qwen/Qwen3-VL-4B-Instruct --local-dir ./playground/Pretrained_models/Qwen3-VL-4B-Instruct 
hf download StarVLA/Qwen3-VL-OFT-Robotwin2 --local-dir ./playground/Pretrained_models/Qwen3-VL-OFT-Robotwin2

export PYTHONPATH=$(pwd):${PYTHONPATH} # let LIBERO find the websocket tools from main repo
export star_vla_python=/usr/bin/python3
your_ckpt=./playground/Pretrained_models/Qwen3-VL-OFT-Robotwin2/checkpoints/steps_40000_pytorch_model.pt
gpu_id=0
port=${PORT:-5694}
echo port: $port
################# star Policy Server ######################

# export DEBUG=true
CUDA_VISIBLE_DEVICES=$gpu_id ${star_vla_python} deployment/model_server/server_policy.py \
    --ckpt_path ${your_ckpt} \
    --port ${port} \
    --use_bf16
```

but get a weird output:

https://github.com/user-attachments/assets/dd26ec6c-6aa9-4be0-9b5c-eaa35c574606

I run this code on L20 GPU, unnorm_key: "robotwin", action_mode: "abs"

Could you tell me why this situation has occurred?

Thanks

**Comment by Axi404** [comment]
We are not sure if this model is still under maintain, but maybe you can try this one: https://huggingface.co/StarVLA/Qwen3-VL-OFT-RoboTwin2-All.

btw, do you replace the path in examples/Robotwin/eval_files/deploy_policy.yml?

**Comment by CuriousCat-7** [comment]
> We are not sure if this model is still under maintain, but maybe you can try this one: https://huggingface.co/StarVLA/Qwen3-VL-OFT-RoboTwin2-All.
> 
> btw, do you replace the path in examples/Robotwin/eval_files/deploy_policy.yml?

Thank you for your reply. But this model still acts wired:

https://github.com/user-attachments/assets/b9aed8b9-7c5d-44d2-b848-eefd11a4f4a1

"btw, do you replace the path in examples/Robotwin/eval_files/deploy_policy.yml?" --> Yes I've replaced the deploy_policy.yml
to 
```yaml
policy_name: starVLA
task_name: null
task_config: null
ckpt_setting: null
seed: null
instruction_type: unseen

# StarVLA Client Parameters
host: "[my_other_machine's ipv6 address]" # because I run this model on H20, and the simulator on L20
port: 9224
policy_ckpt_path: "MY_ROOT/hf_cache/hub/models--StarVLA--Qwen3-VL-OFT-RoboTwin2-All/snapshots/727645249e6bfbff6870db4637e4b9d32e6be346/checkpoints/steps_140000_pytorch_model.pt"
unnorm_key: "robotwin"
action_mode: "abs"
```

**Question I want to confirm:**

Possible reasons I can think of for this behavior:
1. The model was not trained on data generated by an L20 GPU, resulting in very poor generalization.
2. Communication issue: My model runs on an H20 GPU, while the simulator runs on an L20 GPU. The actions of the WebSocket server and client may not be aligned.
3. Was the model trained using absolute joint positions (abs qpos)? Could it have been trained using end-effector (EE) data instead?
4. Normalization issue?

**Comment by JinhuiYE** [comment]
> We are not sure if this model is still under maintain, but maybe you can try this one: https://huggingface.co/StarVLA/Qwen3-VL-OFT-RoboTwin2-All.
> 
> btw, do you replace the path in examples/Robotwin/eval_files/deploy_policy.yml?

 https://huggingface.co/StarVLA/Qwen3-VL-OFT-RoboTwin2 is still working very in this version.

**Comment by cheng-haha** [comment]
@CuriousCat-7 Hi, I also have a similar problem. Have you solved it?

**Comment by CuriousCat-7** [comment]
> [@CuriousCat-7](https://github.com/CuriousCat-7) Hi, I also have a similar problem. Have you solved it?

Unfortunately no, I just used the [X-VLA](https://github.com/2toinf/X-VLA) pre-trained model as the baseline instead, which is also not a perfect solution, but it works for Robotwin.

Also, I think just use the openpi implementation in the Robotwin repo is the shortest way to get a baseline. But if you really want to use starVLA, I suggest you to finetune it by demo_clean data created by yourself.

**Comment by yamanoko-do** [comment]
I encountered a similar issue when trying to reproduce the weights of pre-trained model. Downgrading transformers from version 5.3.0 to 4.57.0 resolved the problem.

**Comment by JinhuiYE** [comment]
Huge thx for sharing this!!!
